articles/arm-x86/index.md

```diff
@@ -0,0 +1,85 @@
+
+There's also an issue of speed. Since ARM has fewer instructions, developers need to use more instructions. For example, did you know that [ARM usually doesn't have a division instruction](https://cseweb.ucsd.edu/classes/wi14/cse30-c/lectures/PI_WI_14_CSE30_lecture_8_post.pdf)? You have to divide by using [other instructions](https://stackoverflow.com/questions/19844575/how-to-do-division-in-arm). Since you're using other instructions to fake division, it ends up taking many cycles. This could even be slower than on a CISC instruction set.
+
+### Executable Sizes
+
+Since you need more instructions in ARM, it's possible that executables will be larger. To test this, I compiled the [sorting_algos repository](https://github.com/botahamec/sorting_algos) made for our [sorting article.](https://www.section.io/engineering-education/sorting-algorithms/) I ran it on my [Raspberry Pi 4 Model B Rev 1.1](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/). I compiled the same thing on my x86-64 Linux desktop. The Raspberry Pi file size should actually be a little smaller. It's only using a 32-bit operating system (Raspberry Pi OS Lite). So, I cross-compiled to the stable-i686-unknown-linux-gnu toolchain on my desktop. "Stripped" means that I stripped nonessential symbols from the executable. The Cargo profiles I used are at the bottom. Here are the results:
+
+|                                  | x86-32  | ARM-32  |
+| -------------------------------- | ------- | ------- |
+| Unoptimized (unstripped)         | 4.39 MB | 4.29 MB |
+| Unoptimized (stripped)           | 595 KB  | 407 KB  |
+| Optimized for speed (unstripped) | 2.71 MB | 2.75 MB |
+| Optimized for speed (stripped)   | 317 KB  | 231 KB  |
+| Optimized for size (unstripped)  | 1.14 MB | 1.13 MB |
+| Optimized for size (stripped)    | 272 KB  | 206 KB  |
+
+Wait, what? Yes, I double-checked. It seems like the ARM binaries are typically smaller. I tried disassembling the binaries. I hope you can forgive me for deciding not to parse thousands of lines of assembly code. It's probably because of a combination of a few different reasons. Here are some ideas:
```

Comment:
Too informal and sounds like a personal blog - reword. Please stick to third-person, factual writing. The engineering education blog is posted on Section's company website and, while it is geared to novice and intermediate programmers, it has to adhere to high professional standards.

Reply:
Synthesize these results a little clearly and what these numbers mean.